SAR_retrieval_funcs.py

```diff
@@ -20,16 +20,16 @@ def wcm_jac(A, V1, B, V2, C, sigma_soil, theta=23):
     often related to canopy moisture content (this is polarisation
     and frequency dependent). The soil backscatter is modelled as
     a linear function of volumetric soil moisture.
-    
-    This function returns the gradient for all parameters (A, B, 
+
+    This function returns the gradient for all parameters (A, B,
     V1, V2 and C)."""
     mu = np.cos(np.deg2rad(theta))
     tau = np.exp(-2 * B * V2 / mu)
-    veg = A * V1 * (1 - tau)
+    veg = A * V1 * np.cos(theta) * (1 - tau)
```

Comment:
I think it'd be better to use something like `mu` instead of `np.cos(theta)` due to the radians/degrees (same goes for Jacobian calculations below)

Reply:
yes you are right